### DISCUSSION TITLE
PNG transparent color becoming black in JPG format

### DISCUSSION THREAD

**Discussion by ralphilius** [dlemstra/magick-wasm#172] [Q&A] [answered]
Here is my code:
```
image.format = targetFormat as MagickFormat;
image.backgroundColor = new MagickColor(255,255,255);
const blob = await image.write(async (data) => {
  return new Blob([data], { type: `image/${targetFormat}` });
});
```
How do I get the transparent part of PNG images to white?
![before](https://github.com/user-attachments/assets/13969a97-0bf3-4d2d-9cd4-419c29fe7dea)
![after](https://github.com/user-attachments/assets/abcefb3f-001c-484c-9e6a-8257908bc298)

**Comment by dlemstra** [comment, marked as answer]
You can use `image.colorAlpha(MagickColors.White)` to do that.

**Reply by ralphilius** [reply]
Thanks. It's working.
